xla_native_functions.yaml

```diff
@@ -164,7 +164,6 @@ supported:
   - kl_div_backward
   - kthvalue
   - l1_loss
```

Comment:
Oh - I'm going to confirm locally, but I think this is why the test was failing. Now that `l1_loss` is `CompositeImplicitAutograd` in core, we should remove XLA's lowering for both `l1_loss_backward` and for `l1_loss`.

Diagnosis: the `l1_loss` entry tells the codegen to register a kernel to the `XLA` backend key for `l1_loss`. Since there isn't a derivative formula for `l1_loss` in core anymore, then the dispatcher registration logic interprets this as "we should ignore autograd" and runs the backend kernel here instead of the autograd-aware decomposition.

Reply:
yup, confirmed locally that the test passes when you also remove the `l1_loss` lowering

Reply:
sg.. let mer verify the HLO generated after it is removed. It is kind of scary to remove them since sometime different HLO choice introduce weird regressions.

Reply:
@Lezcano Sorry for the delay, I will try to get it done tmr.

Reply:
np. Thank you!